src/main/kotlin/no/nav/api/utbetalinger/UtbetalingerService.kt

```diff
@@ -44,4 +48,22 @@ class UtbetalingerService(private val utbetalingerClient: UtbetalingerClient) {
             }
             .sortedWith(compareByDescending<Utbetalinger> { it.til }.thenBy { it.ytelse })
     }
+    
+    suspend fun hentRestUtbetalinger(fnr: String, fra: LocalDate, til: LocalDate): List<Utbetalinger> {
+        val utbetalinger = utbetalingerRestClient.hentUtbetalinger(fnr, fra, til)
+            .also { reporter.reportOk() }
```

Comment:
For å håndtere at det kan ha skjedd feil så tror jeg vi kan håndtere det sånn; 
```suggestion
        val utbetalinger = utbetalingerRestClient.runCatching {
          hentUtbetalinger(fnr, fra, til)
        }
        .onSuccess { reporter.reportOk() }
        .onFailure { reporter.reportError(it) }
        .getOrThrow()
```